src/lib/config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     PATHS: {
       SESSION: "session",
       AUTHORIZATION: "authorization",
+      CHECK: "/check",
```

Comment:
Is the / in front of the check necessary? Doesn't seem to fit with the pattern of session and authorization above

Reply:
Removed!